products/early_access_features/migrations/0001_initial_migration.py

```diff
@@ -0,0 +1,80 @@
+# Generated by Django 4.2.15 on 2024-12-06 14:43
```

Comment:
I’m on the fence on having each product be a separate django app (see [this article](https://careersatdoordash.com/blog/tips-for-building-high-quality-django-apps-at-scale/). Basically Django doesn’t guarantee execution order of migrations I think so it potentially could be a bit of a nightmare. However, not having different product have merge conflicts when doing migrations is definitely a + for productivity so maybe it is worth it

Reply:
In the article they mention fixing it with a homegrown migration manifest: https://gist.github.com/084ef47c4f68922c587bf1252e2d6d89

So they just have a text file with the order in which migrations are run, globally for all apps. We should have this regardless, it'll make merging migration conflicts so much easier -> just move your up/down, wherever.